src/model.ts

```diff
@@ -30,6 +30,8 @@ export class JupyterCadModel implements IJupyterCadModel {
     this.sharedModel.changed.connect(this._onSharedModelChanged);
     this.sharedModel.awareness.on('change', this._onClientStateChanged);
     this.annotationModel = annotationModel;
+
+    this._view = new Map<string, JSONValue>();
```

Comment:
Can we keep `JupyterCadModel` only content related and put the GUI state elsewhere? Maybe we might need another model for the main view to control it from other elements of the interface.

Reply:
Sorry, I don't follow this one. I just take a look at the [NotebookModel](https://github.com/jupyterlab/jupyterlab/blob/d614e293eb4b90055307828d5e81c9661e1147c9/packages/notebook/src/model.ts#L99) of JupyterLab, and I feel like it's only related to the content and the shared model of the notebook.